### ISSUE TITLE
Document trac-post-commit-hook functionality in svn-dev policy

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3867]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3867|
|Type|enhancement|
|Created|2009-06-05 00:08:28Z|
|Branch|https://github.com/twisted/twisted/tree/commit-hooks-3867|

Twisted uses the [trac-post-commit-hook](http://bazaar.launchpad.net/%7Eexarkun/twisted-trac-integration/trunk/annotate/head%3A/trac-hooks/trac-post-commit-hook) to enable ticket updates based on svn commit logs. Would be good to document this in [doc/development/policy/svn-dev.xhtml](doc/development/policy/svn-dev.xhtml) so Twisted developers can take advantage of this functionality. 

<details><summary>Searchable metadata</summary>

```
trac-id__3867 3867
type__enhancement enhancement
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_commit_hooks_3867 branches/commit-hooks-3867
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1244160508000000 1244160508000000
changetime__1264110907000000 1264110907000000
version__None None
owner__jesstess jesstess
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess<br>@jesstess set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

(In [28031]) Branching to 'commit-hooks-3867'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess commented|
|-|-|

(In [28032]) Add section in the SVN development doc about post commit hooks.

refs [#3867](https://github.com/twisted/twisted/issues/3867)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess removed owner|
|-|-|

How does creating my-branch-<patch #> get turned into a trac comment?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

I updated the [link](http://bazaar.launchpad.net/%7Eexarkun/twisted-trac-integration/trunk/annotate/head%3A/trac-hooks/trac-post-commit-hook) in the ticket description, Twisted uses a customized version of this Trac hook afaik.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set owner to @jesstess|
|-|-|

Can you update the link to the commit-hook and maybe use something like:

```
Merge mybranch-123: A single-line summary.
```

or 

```
Apply my.patch: A single-line summary.
```

or both? I think we need a exact example or something..

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_8"></a>|@jesstess commented|
|-|-|

(In [28048]) Link to the custom trac-post-commit-hook and use 1 consistent real example.

refs [#3867](https://github.com/twisted/twisted/issues/3867)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_10"></a>|@khorn set owner to @jesstess|
|-|-|

Looks good.  Merge.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_11"></a>|@jesstess set status to `closed`|
|-|-|

(In [28050]) Merge commit-hooks-3867

Author: jesstess
Reviewers: thijs, khorn
Fixes: [#3867](https://github.com/twisted/twisted/issues/3867)

Add documentation for trac-post-commit-hook functionality in svn-dev.xhtml.
